README.asciidoc

```diff
@@ -404,14 +378,15 @@ hook global WinSetOption filetype=<language> %{
 }
 ----
 
-The faces used for semantic tokens and modifiers can be modified in `kak-lsp.toml`, using the `semantic_tokens.faces` array, for example:
+The faces used for semantic tokens and modifiers can be modified by changing the `lsp_semantic_tokens` option, for example:
```

Comment:
> modifiers can be modified by changing

Three words with the same meaning right next to each other are probably going to trip up people reading this. I'd drop one of them:

```suggestion
The faces used for semantic tokens and modifiers can be changed through the `lsp_semantic_tokens` option, for example:
```